awspub/image.py

```diff
@@ -455,6 +459,10 @@ def create(self) -> Dict[str, _ImageInfo]:
         if self.conf["share"]:
             self._share(self.conf["share"], images)
 
+        if missing_regions:
+            logger.error("Failed to publish images to all regions", extra={"missing_regions": missing_regions})
```

Comment:
will the missing_regions be visible in the log output with this extra param?

Reply:
yes